packages/integration-karma/helpers/test-setup.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+// Global beforeEach/afterEach/etc logic to run before and after each test
+
+var knownChildren;
```

Comment:
Is this to account for the elements created in beforeAll()?

Should we do a similar thing for beforeAll and afterAll too.
For example, `integration-karma/test/component/aom-setter/index.spec.js` uses beforeAll() but does not clean them up. I expect the beforeAll() runs before beforeEach() and creates the elements. These would not be cleaned up by this.
Alternatively, those tests can be fixed to add an afterAll() to clean up the dom.